test/Graphics/StencilMode.test.cpp

```diff
@@ -0,0 +1,299 @@
+#include <SFML/Graphics/Image.hpp>
+#include <SFML/Graphics/RectangleShape.hpp>
+#include <SFML/Graphics/RenderTexture.hpp>
+#include <SFML/Graphics/StencilMode.hpp>
```

Comment:
https://github.com/SFML/SFML/blob/00b6488c3e9350bda573e9a82a06a1d855fa494b/test/Graphics/Texture.test.cpp#L3

Can you please follow the pattern here to ensure StencilMode.hpp is the first header included? This ensures that we're testing that each header can be compiled on its own.

(Side note, CMake 3.26 adds a new feature that lets us test that each header can be compiled on its own but 3.26 is out of reach for the time unless we decide to upgrade our CMake requirement once again. This feature is a lot nicer than having to fight clang-format to ensure the correct header is included first.)